packages/http-client/src/remote-admin-api.ts

```diff
@@ -7,40 +7,57 @@ import { StreamID } from '@ceramicnetwork/streamid'
 export class RemoteAdminApi implements AdminApi {
   // Stored as a member to make it easier to inject a mock in unit tests
   private readonly _fetchJson: typeof fetchJson = fetchJson
+
   readonly baseURL = './admin/models'
 
   constructor(
-    private readonly _apiUrl: URL
+    private readonly _apiUrl: URL,
+    private readonly _context: Context
```

Comment:
jumping in here with a nitpick since it came up in discussion with @ukstv this morning on another PR.  We'd like to move away from the `Context` object as a way to pass dependencies through our stack.  Can you change this to take the DID instance directly instead?

Reply:
Removed `_context`